src/initialize.cc

```diff
@@ -6,10 +6,17 @@
 #include <dmlc/logging.h>
 
 namespace mxnet {
+namespace op {
+void RegisterLegacyOpProp();
+void RegisterLegacyNDFunc();
+}
+
 class LibraryInitializer {
  public:
   LibraryInitializer() {
     dmlc::InitLogging("mxnet");
+    mxnet::op::RegisterLegacyOpProp();
```

Comment:
ok...
How about use this?
http://stackoverflow.com/questions/1113409/attribute-constructor-equivalent-in-vc